package.json

```diff
@@ -111,6 +110,7 @@
     "figures": "^3.2.0",
     "inversify": "^5.0.1",
     "moment": "^2.24.0",
+    "node-fetch": "^2.6.0",
```

Comment:
dont you prefer to stick it to minor only? using ~ ?

Reply:
Assuming everyone adheres to semver, why not stick it to major? Minor changes can introduce bug fixes as well as new functionality so I don't see a reason why it should, in an ideal world, cause any trouble.
Of course I have seen many patch releases that introduced major breaking changes but in those cases even using ~ won't protect you.
Do you have a definitive case against using ^?

Reply:
Bugs usually appears in medium changes, the small ones are very containable.

Reply:
Ok i will restrict all deps